frontend/app-development/layout/SettingsModalButton/SettingsModal/components/Tabs/AccessControlTab/AccessControlTab.test.tsx

```diff
@@ -63,7 +64,100 @@ describe('AccessControlTab', () => {
     expect(screen.getByText(errorMessage)).toBeInTheDocument();
   });
 
-  it('should render all checkboxes as unchecked when applicationMetadata des not contain partyTypes allowed', async () => {
+  it('renders the header', async () => {
+    render();
+    const header = screen.getByRole('heading', {
+      name: textMock('settings_modal.access_control_tab_heading'),
+    });
+    expect(header).toBeInTheDocument();
+  });
+
+  it('should render the button for help text.', async () => {
+    render();
+    const helpButton = screen.getByRole('button', {
+      name: 'helptext',
+    });
+    expect(helpButton).toBeInTheDocument();
+  });
+
+  it('should render the text of the button for help text correctly', async () => {
+    const user = userEvent.setup();
+    render();
+    const helpButton = screen.getByRole('button', { name: 'helptext' });
```

Comment:
`helptext ` used as title in <`Helptext`> should we add it to texts resources? or  maybe remove `name`,and  make it just :
`const helpButton = screen.getByRole('button');`  it works like this.

Reply:
I think you need to replace the 'helptext' in the actual HelpText component with a text from the resources that is more descriptive since that text is used by the screenreaders when they hover over the helptext button. So maybe something like; `Hjelpetekst for oppstartskontroll`